mindsdb/api/http/namespaces/tree.py

```diff
@@ -57,6 +57,17 @@ def get(self, db_name):
                 'engine': val.get('engine'),
                 'deletable': val.get('deletable')
             } for key, val in tables.items()]
+
+            jobs = ca.jobs_controller.get_list(db_name)
+            tables = tables + [{
```

Comment:
wouldn't be better to add list of the jobs into [project.get_tables](https://github.com/mindsdb/mindsdb/blob/3c24a594e8b173404c18cc9ebe679d51a1617165/mindsdb/interfaces/database/projects.py#L305) instead of this one.
get_tables returns objects of the project

Reply:
@ea-rus The GUI uses this endpoint to build a sidebar.